data/i18n/pl/pl.toml

```diff
@@ -142,65 +142,143 @@ other = "adres e-mail"
+<p>Bardzo dziękujemy O'Reilly i Katakodzie za to, że przez wiele lat pomagali naszym użytkownikom w stawianiu ich pierwszych kroków w nauce Kubernetesa.</p>
+<p>Tutoriale przestaną funkcjonować po <b>31 marca 2023 r.</b> Więcej informacji znajdziesz na naszym blogu, zob. <a href="/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">Free Katacoda Kubernetes Tutorials Are Shutting Down</a>.</p>"""
+
 [latest_release]
 other = "Najnowsze wydanie:"
 
 [latest_version]
 other = "najnowszą wersję."
 
-[layouts_blog_pager_prev]
-other = "<< Poprzedni"
-
 [layouts_blog_pager_next]
 other = "Następny >>"
 
+[layouts_blog_pager_prev]
+other = "<< Poprzedni"
+
 [layouts_case_studies_list_tell]
-other = "Opowiedz swoją historię"
+other = "Poznajmy się"
 
 [layouts_docs_glossary_aka]
-other = "Znany też jako"
+other = "Znany także jako"
 
-[layouts_docs_glossary_description]
-other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminologii związanej z projektem Kubernetes. Słownik zawiera terminy specyficzne dla Kubernetesa, a także pojęcia bardziej ogólne, umożliwiające lepsze zrozumienie kontekstu."
+[layout_docs_glossary_architecture_description]
+other = "Komponenty Kubernetesa"
 
-[layouts_docs_glossary_deselect_all]
-other = "Odznacz wszystko"
+[layout_docs_glossary_architecture_name]
+other = "Architektura"
 
 [layouts_docs_glossary_click_details_after]
-other = "po dokładniejsze wytłumaczenie."
+other = "we wskaźnik poniżej, by uzyskać więcej informacji"
 
 [layouts_docs_glossary_click_details_before]
-other = "Kliknij w"
+other = "Kliknij"
+
+[layout_docs_glossary_community_description]
+other = "Dotyczący otwartoźródłowego projektu Kubernetesa"
+
+[layout_docs_glossary_community_name]
+other = "Społeczność"
+
+[layout_docs_glossary_core-object_description]
+other = "Zasób domyślnie obsługiwany przez Kubernetesa."
+
+[layout_docs_glossary_core-object_name]
+other = "Najważniejsze obiekty"
+
+[layouts_docs_glossary_description]
+other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminów i pojęć związanych z projektem Kubernetesa. Słownik zawiera hasła specyficzne dla Kubernetesa, jak i te o bardziej ogólnym charakterze, mające pomóc w lepszym zrozumieniu kontekstu."
+
+[layouts_docs_glossary_deselect_all]
+other = "Odznacz wszystko"
+ 
+[layout_docs_glossary_extension_description]
+other = "Wspierane rozszerzenia Kubernetesa"
 
 [layouts_docs_glossary_filter]
-other = "Znajdź pojęcia według etykiet"
+other = "Szukaj terminów według powiązanych z nimi etykiet"
+
+[layout_docs_glossary_fundamental_description]
+other = "Istotne dla początkujących użytkowników Kubernetesa"
+
+[layout_docs_glossary_fundamental_name]
+other = "Podstawowe"
+
+[layout_docs_glossary_networking_description]
+other = "Jak komponenty Kubernetesa łączą się ze sobą (i z programami spoza klastra)?"
+
+[layout_docs_glossary_networking_description]
+other = "Sieć"
```

Comment:
Netlify build failed.
https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6563c2c9e223d40008bda835#L132
> Error: add site dependencies: load resources: loading translations: "/opt/build/repo/data/i18n/pl/pl.toml:1:1": failed to load translations: toml: table `layout_docs_glossary_networking_description` already exists

Please remove one of duplicate entries.

Reply:
Thanks! Done.